ansible/roles/products/common/local-hand-launch/tasks/main.yml

```diff
@@ -24,22 +24,32 @@
     right_hand_picture: hand_lite.jpg
     left_hand_picture: hand_lite_left.jpg
     bimanual_hands_picture: hand_lite_bimanual.jpg
-  when: product=='hand_lite'
+  when: '"hand_lite" in product'
 
 - name: Set icon picture filenames for hand_extra_lite
   set_fact:
     right_hand_picture: hand_extra_lite.jpg
     left_hand_picture: hand_extra_lite_left.jpg
     bimanual_hands_picture: hand_extra_lite_bimanual.jpg
-  when: product=='hand_extra_lite'
+  when: '"hand_extra_lite" in product'
+
+- name: set product variable
+  set_fact:
+    hand_type_value: "hand_type:=hand_g"
+  when: '"hand_extra_lite" in product or "hand_lite" in product'
+
+- name: set product variable
+  set_fact:
+    hand_type_value: "hand_type:=hand_e"
+  when: 'product is not defined or "hand_e" in product'
 
 - name: Creating server setup script for local right hand launch
   template:
     src: ../../../common/resources/templates/scripts/setup-hand.j2
     dest: "{{ shadow_hand_launcher_folder }}/{{ local_right_setup_script }}"
     mode: '755'
   vars:
-    launch_file_name: "srhand.launch sim:={{ sim_hand }} hand_side_ad:=-r"
+    launch_file_name: "srhand.launch sim:={{ sim_icon }} hand_id:=rh hand_side_ad:=-r {{ hand_type_value }}"
```

Comment:
Before this PR gets merged can you test this on HW? I am wondering what will happen if you have sim_icon:false and hand_type incorrectly defined. What about defining the type only if sim, because autodetection picks up the type automatically.

Reply:
Hand type cannot be incorrectly defined, as we have a validation function that makes sure the product value is valid

Reply:
So it makes sure its a hand_e, hand_lite, hand_extra_lite

Reply:
I mean a case where you run the oneliner for hand_e but connect a hand_lite. I am curious how this will behave :)

Reply:
Oooh I get what you mean :P